frontend-react/cypress/support/commands.js

```diff
@@ -1,78 +1,67 @@
-// ***********************************************
-// This example commands.js shows you how to
-// create various custom commands and overwrite
-// existing commands.
-//
-// For more comprehensive examples of custom
-// commands please read more here:
-// https://on.cypress.io/custom-commands
-// ***********************************************
-//
-//
-// -- This is a parent command --
-// Cypress.Commands.add('login', (email, password) => { ... })
-//
-//
-// -- This is a child command --
-// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
-//
-//
-// -- This is a dual command --
-// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
-//
-//
-// -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 import "@testing-library/cypress/add-commands";
 
-import {OktaAuth} from '@okta/okta-auth-js';
-
 import "cypress-localstorage-commands";
-import {authenticator} from "otplib";
+import { authenticator } from "otplib";
+import { OktaAuth } from "@okta/okta-auth-js";
+
+// this will need to be dynamic as we move to CI
+const REDIRECT_URI = "http://localhost:3000/login/callback";
+const RESPONSE_MODE = "okta_post_message";
+
+const oktaAuthConfig = {
+    issuer: `${Cypress.env("okta_url")}/oauth2/default`,
+    clientId: Cypress.env("okta_client_id"),
+    redirectUri: REDIRECT_URI,
+    responseMode: RESPONSE_MODE,
+    tokenManager: {
+        storage: "sessionStorage",
+    },
+    scopes: ["openid", "email"],
+};
 
 // Okta
-Cypress.Commands.add('loginByOktaApi', (username, password) => {
+Cypress.Commands.add("loginByOktaApi", (username, password) => {
+    // log in with username and password
     cy.request("POST", "https://hhs-prime.oktapreview.com/api/v1/authn", {
         username,
         password,
         options: {
-            multiOptionalFactorEnroll: true,
+            multiOptionalFactorEnroll: false,
```

Comment:
Does this stop the enrollment of 2FA overall or just _multiple_ 2FA methods (i.e. setting up text THEN setting up Authy)? 

(Noting: Aware this should only affect Cypress, not our app's okta rules, but still curious because the name of the property is so weird)

Reply:
my take on this property is whether okta will prompt the user to enroll in mfa, period. It didn't seem to do anything when I switched it up though